integration/ig/k8s/Makefile

```diff
@@ -1,5 +1,7 @@
 include $(shell pwd)/../../../minikube.mk
 
+DNSTESTER_IMAGE ?= ""
```

Comment:
Doesn't it need to be `ghcr.io/inspektor-gadget/dnstester:latest` in case the env variable is not defined?